src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,63 @@
 package edu.kis.vh.nursery;
 
+/**
+ * @author Jan
+ * 
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int STACK_SIZE = 12;
+	private static final int FULL_COUNT = 11;
+	private static final int STACK_EMPTY = -1;
 
-	public int total = -1;
+	private int[] numbers = new int[STACK_SIZE];
 
+	private int total = STACK_EMPTY;
+
+	/**
+	 * @return total elements
+	 */
+	public int getTotal() {
+		return total;
+	}
+
+	/**
+	 * @param in insert in on top of stack
+	 */
 	public void countIn(int in) {
 		if (!isFull())
 			numbers[++total] = in;
 	}
 
+	/**
+	 * @return true if empty
+	 */
 	public boolean callCheck() {
-		return total == -1;
+		return total == STACK_EMPTY;
 	}
 
+	/**
+	 * @return true if full
+	 */
 	public boolean isFull() {
-		return total == 11;
+		return total == FULL_COUNT;
 	}
 
-	protected int peekAboo() {
+	/**
+	 * @return number from top of the stack
+	 */
+	public int peekAboo() {
 		if (callCheck())
-			return -1;
+			return STACK_EMPTY;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)